### PR TITLE
Improve readme training

### PR DESCRIPTION
**Pull request by patrickvonplaten**
Some minor corrections

### CODE REVIEW

File: training/README.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ git lfs install
 git clone https://huggingface.co/sanchit-gandhi/distil-whisper-large-v2-hi
 ```
 
-Be sure to change the repo address to https://huggingface.co/<your-user-name>/<your-repo-name>.
+Be sure to change the repo address to `https://huggingface.co/<your-user-name>/<your-repo-name>`
```
Comment (patrickvonplaten) [inline training/README.md:186]: otherwise it's not shown correctly

File: training/requirements.txt [hunk 2]
```diff
@@ -6,5 +6,4 @@ jiwer
 evaluate>=0.4.1
 wandb
 tensorboard
-tensorflow-cpu
```
Comment (patrickvonplaten) [inline training/requirements.txt:9]: we don't need tensorflow no?

Comment (sanchit-gandhi) [reply training/requirements.txt:9]: Ah yes we can remove! `tensorflow-cpu` is required for Flax's tensorboard logs, but should be ok for PT 👍

**Review by sanchit-gandhi — APPROVED** [review]
Legend thanks @patrickvonplaten

**Pull request merged**